Sign-up form/form.css

```diff
@@ -0,0 +1,99 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    display: grid;
+}
+
+:root {
+    --white: white;
+    --fadedgrey: rgb(238, 238, 238);
+    --green: green;
+}
+
+main {   
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+}
+
+.container {
+    width: 70%;
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+}
+
+/* .odin_logo {
+    position: relative; 
+}  */
+
+.head {
+    background-color: var(--fadedgrey) ;
+}
+
+.form-text {
+    margin-bottom: 30px;
+    font-weight: bold;
+}
+
+.para-text {
+    padding: 0 2rem 2rem;
+}
+
+.para-text > :first-child {
+    margin-bottom: 1rem;
+}
+
+.container > div {
+    /* border: 1px solid red;    */
+}
+
+.head > div {
+    padding-top: 3.3rem;
+}
+
+
+.form-grid {
+    background-color: var(--white);
+    padding: 1rem 2rem;
+    display: flex;
+    box-shadow: 0px 1px 6px -2px #7e7e80;
+}
+
+.grid-1 {
+    margin-bottom: 10px;
+}
+
+
+.grid-2 {
+    margin-left: 2rem;
+}
+
+
+button {
+    background-color: var(--green);
+    color: var(--white);
+    border: 2px solid green;
+    cursor: pointer;
+    width: 120px;
+    margin: 20px 0 0 30px;
+}
+
+.log-in {
+    margin: 10px 0 0 3px;
+    color: var(--green);
+}
+
+.last-content {
+    display: flex;
+    margin: 10px;
+}
+
+.closing-text {
+    margin: 10px 0 50px 20px;
+}
```

Comment:
- Please your project currently has no professional README file. Kindly use the template I provided in the project material
- Please rename your form.html file to index.html. And please push the code for only this form project as I can see folders for recipe and landing page projects here.

Reply:
I'd like to know Sir, I changed the name of the files and after refreshing I didn't see the changes reflected on my github platform. what do I do?

Also, changing the pull request title, does it affect my git branch in any way, as per the name?